spec/workers/enqueue_subject_queue_worker_spec.rb

```diff
@@ -53,19 +72,54 @@
       end
     end
 
-    describe "#load_subjects" do
+    context "when selecting via strategy param" do
+      let(:result_ids) { [1] }
+
+      it 'should fallback from the override strategy' do
+        allow_any_instance_of(Subjects::PostgresqlSelection).to receive(:select).and_return(result_ids)
+        expect(SubjectQueue).to receive(:enqueue)
+        subject.perform(workflow.id, nil, nil, nil, :unknown_strategy)
+      end
+
+      context "with cellect" do
+        let(:run_selection) { subject.perform(workflow.id, nil, nil, nil, :cellect) }
+
+        it 'should attempt to queue the selected set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.